### PR TITLE
FIX: Use volume for fio

### PR DESCRIPTION
**Pull request by julio-santiesteban**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ FROM alpine:3.10.1
 RUN apk update \
   && apk add --no-cache fio==3.13-r1 ca-certificates wget \
   && update-ca-certificates
+VOLUME /app
```
Comment (wgershwitz) [inline Dockerfile]: Did not you mean to provide volume for benchmarking?
If so, we should create a dedicated volume for that and leave app untouched.

**Comment by wgershwitz** [comment]
It might make sense to provide a volume for the jobs too. Alternatively we can use command line arguments too.

File: Dockerfile (hunk 1 above)
Comment (julio-santiesteban) [reply Dockerfile]: Actually, we need a volume for the working directory (where fio does the read/write tests). In this case it happens to be the same directory, but I will create a separate one for it for clarity.

File: run.sh [hunk 2]
```diff
@@ -1,9 +1,10 @@
 #!/bin/sh
 set -eo pipefail
 
-# First remove everything from the folder
-find . -mindepth 1 -maxdepth 1 ! -name 'run.sh' -type f  -exec rm -f {} +
-find . -mindepth 1 ! -name 'jobs' -type d -exec rm -rf {} +
+# First remove everything from the workdir
+rm -rf /data/*
```
Comment (otto-von-bivouac) [inline run.sh]: This is not a good move. If the user attaches an already existing pv/pvc we will wipe it.
Can we please remove it?

File: run.sh [hunk 3]
```diff
@@ -18,7 +19,10 @@ if [ ! -z "$REMOTE_JOB_FILES" ]; then
     done
 fi
 
-# TODO: Parsable JSON output
+# Copy the jobs to the working directory
+cp $JOB_FILES /data
```
Comment (otto-von-bivouac) [inline run.sh]: Would it make sense to create a directory for the benchmark run (using a unique name) and work there?

Comment (otto-von-bivouac) [reply run.sh]: in other words: able to specify the work dir via command line arguments?

File: run.sh (hunk 2 above)
Comment (julio-santiesteban) [reply run.sh]: Removed

File: run.sh (hunk 3 above)
Comment (julio-santiesteban) [reply run.sh]: I don't see the added value of that for such a simple script that usually runs in a container.

File: run.sh (hunk 2 above)
Comment (otto-von-bivouac) [reply run.sh]: it is still there 🙀

File: run.sh (hunk 3 above)
Comment (julio-santiesteban) [reply run.sh]: Added the option anyway :)

Comment (otto-von-bivouac) [reply run.sh]: thank you, very considerate sir!

**Comment by wgershwitz** [comment]
Looks good, thank you!

**Pull request merged**